patches/net/minecraft/server/packs/repository/Pack.java.patch

```diff
@@ -1,21 +1,48 @@
 --- a/net/minecraft/server/packs/repository/Pack.java
 +++ b/net/minecraft/server/packs/repository/Pack.java
-@@ -29,6 +_,7 @@
+@@ -29,6 +_,8 @@
      private final Pack.Position defaultPosition;
      private final boolean required;
      private final boolean fixedPosition;
 +    private final boolean hidden; // Forge: Allow packs to be hidden from the UI entirely
++    private final List<Pack> children; // NeoForge: Allows packs to specify packs which will always be placed beneath them; must be hidden
```

Comment:
Note: the prefix should be `// Neo:` (for both)

Reply:
The former is already fixed in TelepathicGrunt's PR, so when that's merged and I rebase it will be here too - I'll get the new one fixed up